accounts-cluster-bench/src/main.rs

```diff
@@ -340,6 +342,35 @@ fn run_rpc_bench_loop(
             break;
         }
         match rpc_bench {
+            RpcBench::GetAccountInfo => {
+                let mut rpc_time = Measure::start("rpc-get-account-info");
+                let start: u64 = max_closed.load(Ordering::Relaxed);
+                let end: u64 = max_created.load(Ordering::Relaxed);
+                let seed_range = start..end;
+                if seed_range.is_empty() {
+                    info!("get_account_info: No accounts have yet been created; skipping");
+                    continue;
+                }
+                let seed = thread_rng().gen_range(seed_range).to_string();
+                let account_pubkey =
+                    Pubkey::create_with_seed(&base_keypair_pubkey, &seed, &program_id).unwrap();
+                match client.get_account(&account_pubkey) {
```

Comment:
Some percentage of these loads fail with:

```shell
[2024-12-05T22:23:08.309451301Z INFO  solana_accounts_cluster_bench] get_account_info error: Error { request: None, kind: RpcError(ForUser("AccountNotFound: pubkey=Bp88sz54nzCwcxv8S7K6s6Yw5MX4H2FVwVm5sufvBDPa")) }
```

I don't know if `executor.pushTransactions()` is sufficient to presume that the accounts have _actually_ been created.

https://github.com/anza-xyz/agave/blob/master/accounts-cluster-bench/src/main.rs#L643

Is that what's going on here? Is it some sort of race, where I try to get the account before the transaction has committed?

Reply:
Correct, when push_transactions() returns the txs have been queued but not
guaranteed to have been processed and included in a block.

The code that create accounts is _very_ involved, not sure it's easy to change
to guarantee this.